builtin/nomad/platform.go

```diff
@@ -517,6 +529,20 @@ deploy {
 		"The credentials for docker registry.",
 	)
 
+	doc.SetField(
+		"consul_token",
+		"The Consul ACL token used to register services with the Nomad job.",
+		docs.Summary("Uses the runner config environment variable CONSUL_HTTP_TOKEN."),
+		docs.EnvVar("CONSUL_HTTP_TOKEN"),
+	)
+
+	doc.SetField(
+		"vault_token",
+		"The Vault token used to deploy the Nomad job with a token having specific Vault policies attached.",
+		docs.Summary("Uses the runner config environment variable VAULT_TOKEN."),
+		docs.EnvVar("VAULT_TOKEN"),
```

Comment:
Yes! Awesome, thank you. These env var fields are exactly what I was thinking. Looks gooood 👍🏻

Reply:
You might need to run a `make gen/website-mdx. I think this will generate the website docs based on these fields.

Reply:
@briancain done!